api/services/Reddit.js

```diff
@@ -11,6 +11,11 @@ exports.data = {
   adminRefreshToken: sails.config.reddit.adminRefreshToken
 };
 
+/* If 'debug' is set to true, all modifying actions are redirected to /r/crownofnails. This prevents accidental damage to a live sub while debugging.
+This also makes the searchTSVThreads() function only return posts from /r/crownofnails. However, note that removePost() will still remove any post
+sent to it, because it doesn't have a subreddit parameter. */
+var debug = false;
```

Comment:
Huh, clever. Maybe abstract this out to the config file, so that we don't have to change it in code constantly.